data/account.go

```diff
@@ -175,3 +197,80 @@ func (t *AccountTable) Delete(ctx context.Context, projectID uint64, identity pr
 	}
 	return nil
 }
+
+func (t *AccountTable) ListByProjectAndIdentity(ctx context.Context, page Page, projectID uint64, identityType proto.IdentityType, issuer string) ([]*Account, Page, error) {
+	if page.Limit <= 0 {
```

Comment:
we should set upper bound too.. like if page.Limit > 100 then set it to 100 (as example)